### PR TITLE
Check data

### PR DESCRIPTION
**Pull request by hansvancalster**
PR waarin ik getest heb of de GIS en excel data kunnen ingelezen worden. 
Dat lijkt te werken. @RCinbo kan je eens checken of het ook bij u lukt?
Het path naar de data wordt ingesteld via een environmental variable, maar dat zou zichzelf moeten uitwijzen als je de scripts uitvoert.
Ik heb wel op google drive de mapnaam (MoneosMacrozoöbenthos) hernoemt (moneos_benthos). De ö gaf problemen in het path dat in .Renviron ingesteld wordt.

### CODE REVIEW

File: src/scripts/test_inlezen_geodatabank.R [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  geom_sf(data = random_points_2020, colour = "blue", alpha = 0.3)
+
+ecotoop %>%
+  st_drop_geometry() %>%
+  distinct(KRWzone, Naam, Ecotoop, SalZone, NrRandomPunten)
+
+
+ecotoop_aparte_polygonen <- ecotoop %>%
+  select(-starts_with("Shape"), -NrRandomPunten) %>%
+  st_cast("POLYGON")
```
Comment (RCinbo) [inline src/scripts/test_inlezen_geodatabank.R:42]: I received a warning here but I think it's fine...

> In st_cast.sf(., "POLYGON") :
>   repeating attributes for all sub-geometries for which they may not be constant

File: src/scripts/test_inlezen_tabellen.R [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+load_excel <- function(path) {
+  name <- path %>%
+    path_file() %>%
+    path_ext_remove() %>%
+    tolower()
+  assign(name, readxl::read_excel(path), inherits = TRUE)
+}
+
+walk(paths, load_excel)
```
Comment (RCinbo) [inline src/scripts/test_inlezen_tabellen.R:20]: This also gave some warnings but all 4 excels were read correctly.

> Warning messages:
> 1: In UseMethod("depth") :
>   no applicable method for 'depth' applied to an object of class "NULL"
> 2: In read_fun(path = enc2native(normalizePath(path)), sheet_i = sheet,  :
>   Expecting date in C1139 / R1139C3: got '.'
> 3: In read_fun(path = enc2native(normalizePath(path)), sheet_i = sheet,  :
>   Expecting date in C1148 / R1148C3: got '.'

**Review by RCinbo — APPROVED** [review]
Works perfectly! 
Thanks for taking the first step, Hans!

File: src/scripts/test_inlezen_geodatabank.R (hunk 1 above)
Comment (hansvancalster) [reply src/scripts/test_inlezen_geodatabank.R:42]: Yes, that is OK in this case.

File: src/scripts/test_inlezen_tabellen.R (hunk 2 above)
Comment (hansvancalster) [reply src/scripts/test_inlezen_tabellen.R:20]: Yes, I also get these warnings. I actually didn't do any checking of the data. You can consider all these scripts as just drafts to build upon or to recycle snippets of the code in functions in a later phase.

**Pull request merged**